ghost/core/core/server/services/auth/session/session-service.js

```
@@ -275,11 +275,19 @@ module.exports = function createSessionService({
         });
 
         try {
-            await mailer.send({
+            let sendPromise = mailer.send({
                 to: recipient,
                 subject: `${token} is your Ghost sign in verification code`,
                 html: email
-            });
+            })
+            // In Direct mode, it can take 15+ minutes for the mailer to actually fail to send.
```

ESLint

[error] Missing semicolon.

(semi)